.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: CI
+
+on:
+  pull_request:
+  push:
+
+env:
+  RAILS_ENV: test
+  DATABASE_ADAPTER: sqlite3
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  ruby:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '2.7'
+          - '3.0'
+          - '3.1'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: Remove Gemfile.lock
+        run: rm -f Gemfile.lock
```

Comment:
Wouldn't it make sense to run with the lock file? I'm guessing you want to add dependabot for gems too.